testsuite/fuzzer/fuzz/fuzz_targets/move/bytecode_verifier_compiled_modules.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) The Move Contributors
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![no_main]
+use libfuzzer_sys::{fuzz_target, Corpus};
+use move_binary_format::errors::VMError;
+use move_bytecode_verifier::VerifierConfig;
+use move_core_types::vm_status::StatusType;
+mod utils;
+use utils::vm::RunnableState;
+
+fn check_for_invariant_violation_vmerror(e: VMError) {
+    if e.status_type() == StatusType::InvariantViolation
+        // ignore known false positive
+        && !e
+            .message()
+            .is_some_and(|m| m.starts_with("too many type parameters/arguments in the program"))
+    {
+        panic!("invariant violation {:?}", e);
+    }
+}
+
+fuzz_target!(|fuzz_data: RunnableState| -> Corpus {
+    let verifier_config = VerifierConfig::production();
```

Comment:
Can you copy the config from https://github.com/aptos-labs/aptos-core/blob/e7ad67cfaa13127d41be7eb3b889e62efa7a2bf9/aptos-move/aptos-vm-environment/src/prod_configs.rs#L72? this `VerifierConfig::production()` is very bad: it is called "production" but looks nothing like it, especially with new features shipped.